.circleci/config.yml

```
@@ -722,14 +722,11 @@ jobs:
           pattern: '^mysqldb_contrib-.*-mysqlclient'
 
   pymysql:
-    <<: *contrib_job
-    docker:
-      - image: *ddtrace_dev_image
-      - *mysql_server
+    <<: *machine_executor
```

Because this job does not use snapshots, it will not use the testrunnner. This is why we see `/opt/circleci/.pyenv` in the riot logs.

```suggestion
    <<: *contrib_job
```

We could use the pyenv installed on the `ubuntu-2004:current` image we are using for `machine_executor` but they only provide 2.7.18 and 3.10.3:

```
circleci@ip-172-28-22-3:~$ pyenv versions
  system
* 2.7.18 (set by /opt/circleci/.pyenv/version)
* 3.10.3 (set by /opt/circleci/.pyenv/version)
```